jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeModernizerManager.kt

```diff
@@ -136,20 +146,23 @@ class CodeModernizerManager(private val project: Project) : PersistentStateCompo
             return ValidationResult(
                 false,
                 message("codemodernizer.notification.warn.invalid_project.description.reason.invalid_jdk_versions", supportedJavaMappings.keys.joinToString()),
-                InvalidTelemetryReason("JDK", project.tryGetJdk().toString()),
+                InvalidTelemetryReason(
+                    CodeTransformPreValidationError.`Project selected is not Java 8 or Java 11`,
+                    project.tryGetJdk().toString()
+                )
             )
         }
         val valid = getSupportedBuildFilesInProject().isNotEmpty()
         return if (valid) {
```

Comment:
Was a stale metrics that wasn't used anymore. We changed the functionality of the "transform" button to ALWAYS be showing.

This codepath is logged  by `userDoubleClick....`